packages/components/src/CollapsiblePanel/CollapsiblePanel.scss

```diff
@@ -145,3 +169,88 @@ $tc-collapsible-panel-btn-color: #555964;
 		}
 	}
 }
+
+.descriptive-panel {
+	border-radius: 0;
+	margin-bottom: 0;
+
+	:global(.panel-heading) {
+		background-color: white;
+		border-color: transparent;
+	}
+
+	:global(.panel-title) {
+		a {
+			> button {
+				margin: 0;
+				padding: 0 5px;
+				text-decoration: none;
+			}
+		}
+	}
+
+	.group {
+		justify-content: flex-start;
+		display: flex;
+	}
+
+	.title {
+		color: $tc-collapsible-selectable-panel-title-color;
+		font-weight: 600;
+		overflow: hidden;
+		text-overflow: ellipsis;
+		width: 70px;
+		white-space: nowrap;
+	}
+
+	.tag {
+		color: $tc-collapsible-selectable-panel-content-description-color;
+		margin-left: 0;
+		white-space: nowrap;
+	}
+
+	.detail {
+		color: $tc-collapsible-selectable-panel-title-color;
+		white-space: nowrap;
+		justify-content: flex-end;
+	}
+
+	:global(.panel-body) {
+		padding: 0 10px 10px 10px;
+
+		.content {
+			display: flex;
+			flex-direction: column;
+
+			.upper {
+				display: flex;
+				justify-content: space-between;
+
+				> span {
+					overflow: hidden;
+					text-overflow: ellipsis;
+					max-width: 150px;
+					white-space: nowrap;
+				}
+			}
+
+			.content-description {
+				color: $tc-collapsible-selectable-panel-content-description-color;
+			}
+		}
+	}
+
+	&.tc-descriptive-no-content {
+		:global(.panel-heading) {
+			padding: 0 5px 0 0;
+
+			> button {
+				display: flex;
```

Comment:
Here is the result with Firefow 51 (which we have to support)

![win10_firefox_51 0](https://cloud.githubusercontent.com/assets/18534166/24045369/27d65b0c-0b1f-11e7-8720-7c6023ad69da.jpg)